### ISSUE TITLE
ArgumentNullException when running nunit3-console

### ISSUE THREAD

**Issue by marklam** [SteveGilham/altcover#12]
It looks like the ``DataReceivedEventArgs.Data`` property can be null. The [sample code](https://msdn.microsoft.com/en-us/library/system.diagnostics.process.errordatareceived) in msdn has a ``String.IsNullOrEmpty`` test.

The stack trace was:
```
Unhandled Exception: System.ArgumentNullException: Parameter "message" cannot be null.
   at Microsoft.Build.Shared.ErrorUtilities.VerifyThrowArgumentNull(Object parameter, String parameterName, String resourceName)
   at Microsoft.Build.Utilities.TaskLoggingHelper.LogMessage(MessageImportance importance, String message, Object[] messageArgs)
   at AltCover.Collect.Message(String x)
   at <StartupCode$AltCover>.$Tasks.Execute@95-3.Invoke(String arg00)
   at Microsoft.FSharp.Control.CommonExtensions.SubscribeToObservable@1741.System-IObserver`1-OnNext(T args)
   at Microsoft.FSharp.Core.CompilerServices.RuntimeHelpers.h@349.Invoke(Object _arg1, TArgs args)
   at AltCover.CommandLine.Launch@102-8.Invoke(Object delegateArg0, DataReceivedEventArgs delegateArg1)
   at System.Diagnostics.Process.OutputReadNotifyUser(String data)
   at System.Diagnostics.AsyncStreamReader.FlushMessageQueue()
   at System.Diagnostics.AsyncStreamReader.ReadBuffer(IAsyncResult ar)
   at System.IO.Stream.ReadWriteTask.InvokeAsyncCallback(Object completedTask)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.IO.Stream.ReadWriteTask.System.Threading.Tasks.ITaskCompletionAction.Invoke(Task completingTask)
   at System.Threading.Tasks.Task.FinishContinuations()
   at System.Threading.Tasks.Task.FinishStageThree()
   at System.Threading.Tasks.Task.FinishStageTwo()
   at System.Threading.Tasks.Task.Finish(Boolean bUserDelegateExecuted)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot)
   at System.Threading.Tasks.Task.ExecuteEntry(Boolean bPreventDoubleExecution)
   at System.Threading.Tasks.Task.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
   at System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()

Unhandled Exception: System.ArgumentNullException: Parameter "message" cannot be null.
   at Microsoft.Build.Shared.ErrorUtilities.VerifyThrowArgumentNull(Object parameter, String parameterName, String resourceName)
   at Microsoft.Build.Utilities.TaskLoggingHelper.LogError(String subcategory, String errorCode, String helpKeyword, String file, Int32 lineNumber, Int32 columnNumber, Int32 endLineNumber, Int32 endColumnNumber, String message, Object[] messageArgs)
   at Microsoft.Build.Utilities.TaskLoggingHelper.LogError(String message, Object[] messageArgs)
   at <StartupCode$AltCover>.$Tasks.Execute@94-2.Invoke(String arg00)
   at Microsoft.FSharp.Control.CommonExtensions.SubscribeToObservable@1741.System-IObserver`1-OnNext(T args)
   at Microsoft.FSharp.Core.CompilerServices.RuntimeHelpers.h@349.Invoke(Object _arg1, TArgs args)
   at AltCover.CommandLine.Launch@101-3.Invoke(Object delegateArg0, DataReceivedEventArgs delegateArg1)
   at System.Diagnostics.Process.ErrorReadNotifyUser(String data)
   at System.Diagnostics.AsyncStreamReader.FlushMessageQueue()
   at System.Diagnostics.AsyncStreamReader.ReadBuffer(IAsyncResult ar)
   at System.IO.Stream.ReadWriteTask.InvokeAsyncCallback(Object completedTask)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.IO.Stream.ReadWriteTask.System.Threading.Tasks.ITaskCompletionAction.Invoke(Task completingTask)
   at System.Threading.Tasks.Task.FinishContinuations()
   at System.Threading.Tasks.Task.FinishStageThree()
   at System.Threading.Tasks.Task.FinishStageTwo()
   at System.Threading.Tasks.Task.Finish(Boolean bUserDelegateExecuted)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot)
   at System.Threading.Tasks.Task.ExecuteEntry(Boolean bPreventDoubleExecution)
   at System.Threading.Tasks.Task.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
   at System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()


```

**Comment by SteveGilham** [comment]
That one was easy to address.  Fix in [pre-release build 431](https://ci.appveyor.com/project/SteveGilham/altcover/build/3.0.431-pre/artifacts)

**Comment by SteveGilham** [comment]
Fix is in [the v3.0.433 release](https://www.nuget.org/packages/AltCover/3.0.433).
